src/Microsoft.AspNet.Mvc.DataAnnotations/ValidationAttributeAdapterOfTAttribute.cs

```diff
@@ -45,24 +45,22 @@ public TAttribute Attribute
         /// <param name="modelMetadata">The <see cref="ModelMetadata"/> associated with the model annotated with
         /// <see cref="Attribute"/>.</param>
         /// <returns>Formatted error string.</returns>
-        protected virtual string GetErrorMessage(ModelMetadata modelMetadata)
+        protected virtual string GetErrorMessage(ModelMetadata modelMetadata, params object[] arguments)
```

Comment:
Does any caller use the `params` capability? If not or there's only one, remove and make the array allocation explicit in the callers.